src/main/java/com/rcv/ResultsWriter.java

```diff
@@ -702,10 +702,13 @@ void generateCdfJson(List<CastVoteRecord> castVoteRecords)
           entry("OtherType", "vendor-label"),
           entry("Value", config.getNameForCandidateCode(candidateCode))
       );
+      // codeMap is wrapped in an array
+      ArrayList<Map<String, String>> codeArray = new ArrayList<>();
+      codeArray.add(codeMap);
       contestSelections.add(Map.ofEntries(
           entry("@id", getCdfIdForCandidateCode(candidateCode)),
           entry("@type", "CVR.ContestSelection"),
-          entry("Code", codeMap)
+          entry("Code", codeArray)
```

Comment:
this is simpler and matches what I've done elsewhere:

`entry("Code", new Map[]{codeMap}),`

Reply:
👍